files/helpers/strings.py

```diff
@@ -0,0 +1,4 @@
+
+# clean strings for searching
+def clean_string(my_str):
+	return my_str.replace(r'\\', '').replace('_', r'\_').replace('%', '').strip()
```

Comment:
not sure exactly what this is for but it was copy pasted a lot and seems to be related to querying the DB

Reply:
Snakes says this is for `ilike`-match sanitization. Rename this to something like `sql_ilike_clean()`?

Reply:
cool, done